src/vs/base/browser/ui/dialog/dialog.ts

```diff
@@ -82,7 +82,7 @@ export class Dialog extends Disposable {
 		this.focusToReturn = document.activeElement as HTMLElement;
 
 		return new Promise<number>((resolve) => {
-			if (!this.element || !this.buttonsContainer || !this.iconElement || !this.toolbarContainer) {
```

Comment:
Please revert these changes too. Again, this PR should only have spelling fixes and not any other refactorings